src/perfrecord.cpp

```diff
@@ -162,7 +167,36 @@ void PerfRecord::startRecording(bool elevatePrivileges, const QStringList& perfO
                     }
                 });
 
+#ifdef USE_KAUTH
+        KAuth::Action action(QLatin1String("com.kdab.hotspot.perf.elevate"));
```

Comment:
here and below: I bet you are using `QL1S` wrongly, this should be `QStringLiteral` everywhere. generally, please acquaint yourself with the rules there - only use `QL1S` with overloads that support it, but don't use it when a `QString` is expected - then use `QStringLiteral` instead